ddtrace/internal/datadog/profiling/stack_v2/src/stack_v2.cpp

```diff
@@ -0,0 +1,76 @@
+#include "cast_to_pyfunc.hpp"
+#include "python_headers.hpp"
+#include "sampler.hpp"
+
+#include <iostream>
+
+using namespace Datadog;
+
+static PyObject*
+_stack_v2_start(PyObject* self, PyObject* args, PyObject* kwargs)
+{
+    (void)self;
+    std::cout << "Starting the sampler" << std::endl;
+    static const char* const_kwlist[] = { "min_interval", NULL };
+    static char** kwlist = const_cast<char**>(const_kwlist);
+    double min_interval_s = g_default_sampling_period_s;
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|d", kwlist, &min_interval_s)) {
+        return NULL; // If an error occurs during argument parsing
+    }
+
+    Sampler::get().set_interval(min_interval_s);
+    Sampler::get().start();
+
+    Py_INCREF(Py_None);
+    return Py_None;
+}
+
+// Bypasses the old-style cast warning with an unchecked helper function
+PyCFunction stack_v2_start = cast_to_pycfunction(_stack_v2_start);
+
+static PyObject*
+stack_v2_stop(PyObject* self, PyObject* args)
+{
+    (void)self;
+    (void)args;
+    Sampler::get().stop();
+    Py_INCREF(Py_None);
+    return Py_None;
```

Comment:
```suggestion
    Py_RETURN_NONE
```
https://docs.python.org/3.10/c-api/none.html#c.Py_RETURN_NONE